src/app/zwiazek/kalendarz/GenerateCalendar.tsx

```diff
@@ -0,0 +1,24 @@
+import { formatDate } from "@/utils";
+
+export const GenerateCalendar = (date: Date) => {
```

Comment:
Please name functions in a `camelCase` fashion.

No need to have it in a separate file, although it could be abstracted to some `/_utils` within the page module